app/controllers/api/v1/items/search_controller.rb

```diff
@@ -1,7 +1,12 @@
 class Api::V1::Items::SearchController < ApplicationController
   def show
     results = search_results(params).first
-    render json: ItemSerializer.new(results).serialized_json
```

Comment:
Removed serialized_json becuase that isn't necessary for the fast_jsonapi gem we used